easybuild/easyconfigs/g/GenomeWorks/GenomeWorks-2021-02-2_dep_versions_n_file_extension.patch

```diff
@@ -0,0 +1,46 @@
+# author: Denis Kristak (INUITS)
+# set correct dependecy versions && expect *.a files to copy, rather than *.so
+diff -ruN GenomeWorks-dev_orig/pygenomeworks/requirements.txt GenomeWorks-dev/pygenomeworks/requirements.txt
+--- GenomeWorks-dev_orig/pygenomeworks/requirements.txt	2021-06-08 19:42:06.199086912 +0200
++++ GenomeWorks-dev/pygenomeworks/requirements.txt	2021-06-08 19:41:42.299168164 +0200
+@@ -16,12 +16,12 @@
+ 
+ 
+ 
+-Cython~=0.29.14
++Cython==0.29.21
+ intervaltree==3.1.0
+-matplotlib==3.3.2
+-networkx==2.4
+-numpy~=1.17.5
+-pytest==4.4.1
++matplotlib==3.3.3
++networkx==2.5.1
++numpy==1.19.4
++pytest==6.2.4
+ setuptools>=41.4.0
+-sortedcollections==1.1.2
+-tqdm~=4.31
++sortedcollections==2.1.0
++tqdm==4.56.2
+diff -ruN GenomeWorks-dev_orig/pygenomeworks/setup.py GenomeWorks-dev/pygenomeworks/setup.py
+--- GenomeWorks-dev_orig/pygenomeworks/setup.py	2021-06-08 14:59:57.511155059 +0200
++++ GenomeWorks-dev/pygenomeworks/setup.py	2021-06-08 15:00:39.418965132 +0200
+@@ -57,7 +57,7 @@
+     return requirements_file_content
+ 
+ 
+-def copy_all_files_in_directory(src, dest, file_ext="*.so"):
++def copy_all_files_in_directory(src, dest, file_ext="*.a"):
```

Comment:
Hmm, doesn't this mean we're missing an `--enable-shared` configuration option in the easyconfig file, or something like that?

It's possible that having static libraries (`*.a`) is even a real problem in combination with Python, which does a lot of dynamic loading (which required shared libraries (`*.so`) I think)

Reply:
Fixed by adding `-Dgw_build_shared=ON` to `configopts` so shared libraries are built, and then this patch file is no longer needed \o/